pkg/sqlite/query.go

```diff
@@ -541,3 +540,104 @@ func (s *SQLQuerier) GetApisForEntry(ctx context.Context, entryId int64) (provid
 
 	return
 }
+
+func (s *SQLQuerier) GetBundleVersion(ctx context.Context, image string) (string, error) {
+	query := `SELECT version FROM operatorbundle WHERE bundlepath=? LIMIT 1`
+	rows, err := s.db.QueryContext(ctx, query, image)
+	if err != nil {
+		return "", err
+	}
+	defer rows.Close()
+
+	var version sql.NullString
+	if rows.Next() {
+		if err := rows.Scan(&version); err != nil {
+			return "", err
+		}
+	}
+	if version.Valid {
+		return version.String, nil
+	}
+	return "", fmt.Errorf("bundle %s not found", image)
+}
+
+func (s *SQLQuerier) GetImagesForPackage(ctx context.Context, pkgName string) ([]string, error) {
```

Comment:
Where is this used?

Isn't the field called `bundlepath`, not `image`?

Reply:
Good point - I'll change the name